admin/code/LeftAndMain.php

```diff
@@ -444,6 +444,9 @@ public function init() {
 		// The user's theme shouldn't affect the CMS, if, for example, they have
 		// replaced TableListField.ss or Form.ss.
 		Config::inst()->update('SSViewer', 'theme_enabled', false);
+
+		//set the reading mode for the admin to stage
+		Versioned::set_reading_mode('Stage');
```

Comment:
`Versioned::reading_stage('Stage');` not `Versioned::set_reading_mode('Stage');`

You could do `Versioned::reading_stage('Stage.Stage');` too